pages/apply/start.tsx

```diff
@@ -41,19 +34,12 @@ const ApplicationStartPage = (): JSX.Element => {
             nationalInsuranceNumber: values.nationalInsuranceNumber,
```

Comment:
Think `updateBeforeFirstSave` might need to be updated to `updateApplicant`

Reply:
I've put this in but haven't been able to run a true new application. We might need to ensure that the main applicant's id is available at this stage.